app/src/test/java/ch/hikemate/app/model/profile/ProfileRepositoryFirestoreTest.kt

```diff
@@ -75,20 +113,7 @@ class ProfileRepositoryFirestoreTest {
   }
 
   @Test
-  fun documentToProfile_returnsNullIfDocumentIsMissingFields() {
-    `when`(mockDocumentSnapshot.id).thenReturn("1")
-    `when`(mockDocumentSnapshot.getString("name")).thenReturn(null)
-    `when`(mockDocumentSnapshot.getString("email")).thenReturn("john.doe@gmail.com")
-    `when`(mockDocumentSnapshot.getString("hikingLevel")).thenReturn("INTERMEDIATE")
-    `when`(mockDocumentSnapshot.getTimestamp("joinedDate")).thenReturn(Timestamp(1609459200, 0))
-
-    val profile = repository.documentToProfile(mockDocumentSnapshot)
-
-    assert(profile == null)
-  }
```

Comment:
Question: Is there a reason to remove this ?

Reply:
Yes, this function had to be changed it now won't return null whenver there are missing fields. So yes it doesn't make any sense there.